Makefile

```diff
@@ -3,20 +3,19 @@
 REGISTRY ?= quay.io
 REPOSITORY ?= $(REGISTRY)/kohlstechnology
 
-BUILD_COMMIT := $(shell ./scripts/build/get-build-commit.sh)
-BUILD_TIMESTAMP := $(shell ./scripts/build/get-build-timestamp.sh)
-BUILD_HOSTNAME := $(shell ./scripts/build/get-build-hostname.sh)
+COMMIT=`git rev-parse HEAD`
```

Comment:
the script had a feature for adding a `-dirty` suffix; would we want to try and keep it?

Reply:
My opinion is no I don't think we need to add a `-dirty` suffix. Let me know if you think this would still be a useful feature.

Reply:
Hm; now that I think of it, it's too far from the place where it's used anyway, and some commands could be run between the two places, so it could be misleading anyway. So in the end I'm ok with removing it, thanks.